### ISSUE TITLE
Not able to use arrows to navigate hotgrid items

### ISSUE THREAD

**Issue by joe-allen-89** [cgkineo/adapt-hotgrid#107]
### Subject of the issue/enhancement/features
When navigating the hotgrid using a keyboard you're unable to use the arrows to navigate as you could with previous versions. Also when closing a popup the screen reader loses focus and the user is still not able to navigate with arrows it will scroll the page instead. 

Seems to be linked to aria-haspopup.

### Your environment
* v5.24.8
* chrome + Jaws 2023
